crates/esthri-cli/tests/integration/http_server.rs

```diff
@@ -100,6 +100,21 @@ fn upload_test_data() -> anyhow::Result<()> {
     Ok(())
 }
 
+async fn upload_index_url_test_data() -> anyhow::Result<()> {
```

Comment:
Please add comment describing what the test aims to validate

Reply:
this isn't a test, just a helper function for populating test data